capa/features/extractors/common.py

```diff
@@ -36,14 +36,14 @@
 MATCH_RESULT = b'{"meta":'
 
 
-def extract_file_strings(buf, **kwargs) -> Iterator[Tuple[String, Address]]:
+def extract_file_strings(buf, len, **kwargs) -> Iterator[Tuple[String, Address]]:
```

Comment:
lets call this parameter `min_len`

Reply:
suggestions about renaming apply to all relevant files/extractors, so please apply throughout.